packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,75 @@
+import { html, LitElement } from 'lit-element';
+import zendeskWidgetCSS from './styles/zendesk-widget';
+import questionIcon from './icon-question-mark';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get styles() {
+    return zendeskWidgetCSS;
+  }
+
+  static get properties() {
+    return {
+      widgetSrc: { type: String },
+      widgetkey: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.widgetSrc = '';
+    this.widgetkey = '';
+  }
+
+  initiateZenDesk() {
+    // Hide custom button
+    this.style.opacity = 0;
+
+    // prevent further if third party script already loaded
+    if (this.abortIfScriptAlreadyLoaded()) return;
+
+    // Inject help widget
+    this.injectScript();
+
+    let countdownTicksLeft = 20;
+    const timeoutTimer = setInterval(() => {
+      // See if button has appeared
+      const iframe = document.getElementById('launcher');
+      if (iframe) {
+        const button = iframe.contentWindow.document.querySelector('button');
+        if (button) {
+          // Found it! Click on it and hope the window opens!
+          button.click();
+          clearInterval(timeoutTimer);
+          return;
+        }
+      }
+
+      // Try again later
+      countdownTicksLeft--;
+      if (countdownTicksLeft <= 0) clearInterval(timeoutTimer);
+    }, 250);
+  }
+
+  injectScript() {
+    const script = document.createElement('script');
+    script.setAttribute('id', 'ze-snippet');
+    script.setAttribute('src', this.widgetSrc + this.widgetkey);
+    document.head.appendChild(script);    
+  }
+
+  abortIfScriptAlreadyLoaded() {
+    if (document.querySelector(`script[src="${this.widgetSrc + this.widgetkey}"]`)) return;
+  }
+
+  render() {
+    return html`
+      <div
```

Comment:
Since this is a button, I think it'd make more sense semantically to use a `button` tag.